WordPress/src/test/java/org/wordpress/android/viewmodel/pages/PagesViewModelTest.kt

```diff
@@ -183,6 +188,26 @@ class PagesViewModelTest {
         assertThat(viewModel.arePageActionsEnabled).isTrue()
     }
 
+    @Test
+    fun `Auto-upload is initiated when the user enters the screen`() {
+        // Given
+        // When
+        viewModel.start(site)
+        // Then
+        verify(uploadStarter).queueUploadFromSite(site)
```

Comment:
I noticed was when I run this test it passes but there is an exception because the page store is being triggered and it's not set up so we could call `setUpPageStoreWithEmptyPages()` in both tests so the exception is no longer shown. Wdyt?

Reply:
Ohh, I missed that. Great idea! Thanks!
Fixed in 143453475e547212b89d1991f7ecf1cca98e2bec